modules/core/src/test/java/org/apache/ignite/internal/commandline/CommandHandlerParsingTest.java

```diff
@@ -396,6 +397,12 @@ public void testParseAutoConfirmationFlag() {
                     break;
                 }
 
+                case TRACING_CONFIGURATION: {
+                    // TODO: 05.05.20
```

Comment:
It would be nice to have a Jira ticket to address this )

Reply:
Cause we don't have auto confirmation flag it's possible to remove it.